frontend/app/src/main/java/de/amosproj3/ziofa/MainActivity.kt

```diff
@@ -0,0 +1,38 @@
+package de.amosproj3.ziofa
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.activity.enableEdgeToEdge
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import de.amosproj3.ziofa.ui.theme.ZIOFATheme
+
+class MainActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        enableEdgeToEdge()
+        setContent {
+            ZIOFATheme {
+                Scaffold(modifier = Modifier.fillMaxSize()) { innerPadding ->
+                    Greeting(
+                        name = "AMOS",
+                        modifier = Modifier.padding(innerPadding)
+                    )
+                }
+            }
+        }
+    }
```

Comment:
`KoinAndroidContext` is missing (https://insert-koin.io/docs/quickstart/android-compose/#start-koin).

This is not relevant for this commit, but might be confusing in the future.

Reply:
done, Koin Compose library was also missing